Detectors/TRD/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@
 # In applying this license CERN does not waive the privileges and immunities
 # granted to it by virtue of its status as an Intergovernmental Organization
 # or submit itself to any jurisdiction.
+add_compile_options(-O0 -g -fPIC)
```

Comment:
Should be removed - maybe this makes it slow?

Reply:
only now do i not need it anymore, sure of course before finalising this will come out

Reply:
removed